### DISCUSSION TITLE
Navigation Transitions only for content

### DISCUSSION THREAD

**Discussion by sohcah** [bdlukaa/fluent_ui#157] [Q&A]
Hey!

I'm wondering if it's possible to make the Navigation transitions only apply to the content of a page, rather than also to the AppBar and Navigation Page, as it looks rather weird otherwise?
I feel like I'm probably missing something really obvious, but not really sure.

**Comment by bdlukaa** [comment]
Hello! Currently, there is not a way to do that

**Reply by sohcah** [reply]
Ah, fair enough. That's unfortunate, as the default animations for the Navigator are quite jarring when the sidebar moves along with the page. I'll probably end up making my own Navigator to do this... and if so I'll see about open-sourcing it.
